src/gui/egui_gui.rs

```diff
@@ -82,6 +82,7 @@ impl GUI {
                                 key: kind.into(),
                                 pressed: true,
                                 modifiers: modifiers.into(),
+                                repeat: false,
```

Comment:
What did you mean with "TODO: fix after winit 0.28 upgrade", according to the [docs](https://docs.rs/egui/latest/egui/enum.Event.html#variant.Key.field.repeat), it should always be false

Reply:
I read the docs wrong. I thought they wanted you to set the repeat as false *if* your integration doesn't have support for key repeat events. 

It is confusing though, why add a repeat field if you are always going to set it to false?

Reply:
It's only if you make an integration that you have to set it to false, egui will set the field to something else and the user of the library will read the field 🙂